starters/deno-oak-denodb/tools/generateTypeDefinition.ts

```diff
@@ -0,0 +1,21 @@
+import { CodegenConfig, generate } from 'npm:@graphql-codegen/cli';
```

Comment:
Shouldn't this file be named `generate_type_definition.ts` according to the styleguide?

Reply:
I keep spelling that wrong😑